libbeat/docs/communitydashboards.asciidoc

```diff
@@ -0,0 +1,29 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content appears in both the Beats Platform Reference and the
+//// Beats Developer Guide.
+//////////////////////////////////////////////////////////////////////////
+
+[[community-dashboards]]
+== Community Beats
+
+The open source community has been hard at work developing new Dashboards. You can check
+out some of them here. For information on exporting and importing dashboards see the documentation for
+https://www.elastic.co/guide/en/kibana/current/managing-saved-objects.html[Kibana Saved Objects].
```

Comment:
As a best practice, use [shared attributes](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc) to resolve doc links so they don't break over time as content moves around or gets deleted:

```
{kibana-ref}/managing-saved-objects.html[Kibana Saved Objects]